server/auth/github.js

```diff
@@ -0,0 +1,199 @@
+// @flow
+import passport from "@outlinewiki/koa-passport";
+import Router from "koa-router";
+import { Strategy as GithubStrategy } from "passport-github2";
+import accountProvisioner from "../commands/accountProvisioner";
+import env from "../env";
+import * as Github from "../github";
+import auth from "../middlewares/authentication";
+import passportMiddleware from "../middlewares/passport";
+import { Authentication, Collection, Integration, Team } from "../models";
+import { StateStore } from "../utils/passport";
+
+const router = new Router();
+const providerName = "github";
+const GITHUB_CLIENT_ID = process.env.GITHUB_KEY;
```

Comment:
Let's have the `env` variables match (`GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET`), it looks like you copied the Slack integration here but it's not a great example because those are so old and we've had to maintain backwards compatibility.